.github/workflows/build-nightly.yml

```diff
@@ -0,0 +1,153 @@
+name: Build (Nightly)
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - develop
+      - develop-pr-*
+    paths:
+      - "**/*.go"
+      - "**/*.c"
+      - "**/*.h"
+      - "go.mod"
+      - "go.sum"
+      - ".github/workflows/build-nightly.yml"
+      - "Makefile"
+  release:
+    types: [ prereleased ]
```

Comment:
We can only trigger `main` branch commits instead of releasing.

Reply:
Sure, will remove this part accordingly. Resolved in [200526a](https://github.com/daeuniverse/dae/pull/143/commits/200526aa4e8ee4f7c3b28798c898dcb22d37e3ea)